CONTRIBUTING.md

```diff
@@ -0,0 +1,42 @@
+# How to contribute
+
+So, you've decided to contribute, that's great!
+
+You can use this document to figure out how and where to start.
+
+## Getting started
+
+- Make sure you have a [GitHub account](https://github.com/join).
+- Take a look at [existing issues](https://github.com/launchbadge/sqlx/issues).
+- If you need to create an issue:
+  - Make sure to clearly describe it.
+  - Including steps to reproduce when it is a bug.
+  - Include the version of SQLx used.
```

Comment:
We should suggest a mention of what database driver they're using and what version of the database they're using.

Reply:
Good point! Added the extra steps 👍